mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -121,24 +117,14 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
    * Update value
    */
   const updateValue = (updatedValue: T) => {
-    const isExceedingMinLength = typeof updatedValue === "string" &&
-    props.minLength && updatedValue.length < props.minLength;
-
-    const isExceedingMaxLength = typeof updatedValue === "string" &&
-    props.maxLength && updatedValue.length > props.maxLength;
-
-    const isInvalidTextLength = isExceedingMinLength || isExceedingMaxLength;
-
-    const notificationText = isExceedingMinLength
-      ? "Label should include at least one character"
-      : "Label should not exceed 30 characters";
-
-    isInvalidTextLength
-      ? displayNotification({
-        text: notificationText,
-        type: NotificationType.INFO,
-      })
-      : setValue(updatedValue);
+    if (!props.validators) {
+      setValue(updatedValue);
+    } else {
+      if (typeof updatedValue === "string") {
+        const error = validateValue({value: updatedValue, validators: props.validators});
+        error ? displayValidationError?.(error) : setValue(updatedValue);
+      }
+    }
```

Comment:
we can create separate updateValueWithValidatorsHamdler and reuse it in different componsnts

Reply:
you cant use displayValidationError inside (and get rid of displayValidationError itself and use just it body)

Reply:
done